src/Type/Php/MbFunctionsReturnTypeExtension.php

```diff
@@ -86,9 +88,14 @@ public function getTypeFromFunctionCall(FunctionReflection $functionReflection,
 		}
 
 		if (count($results) === 1) {
+			$invalidEncodingReturn = new ConstantBooleanType(false);
+			if ($this->phpVersion->getVersionId() >= 80000) {
```

Comment:
This needs a descriptive method instead. Check the class if there's already a method that would match the situation, otherwise implement one :)

Reply:
implemented `PhpVersion->throwsOnInvalidMbStringEncoding() `